metropolia-university-of-applied-sciences-harvard.csl

```diff
@@ -0,0 +1,440 @@
+      <if variable="title" match="none"/>
+      <else-if type="thesis"/>
+      <else>
+        <text variable="genre" text-case="capitalize-first"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="issue">
+    <choose>
+      <if type="legal_case">
+        <text variable="authority"/>
+      </if>
+      <else-if type="speech">
+        <group prefix=" " delimiter=", ">
+          <text macro="event"/>
+          <text macro="day-month"/>
+          <text variable="event-place"/>
+        </group>
+      </else-if>
+      <else-if type="article-newspaper article-magazine" match="any">
+        <text macro="day-month"/>
+      </else-if>
+      <else>
+        <group prefix=" " delimiter=", ">
+          <choose>
+            <if type="thesis">
+              <text variable="genre" text-case="capitalize-first"/>
+            </if>
+          </choose>
+          <text macro="publisher" prefix=" "/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="note">
+    <text variable="note" prefix="(" suffix=")"/>
+  </macro>
+  <citation et-al-subsequent-min="3" et-al-subsequent-use-first="1" disambiguate-add-names="true" disambiguate-add-givenname="true" disambiguate-add-year-suffix="true" givenname-disambiguation-rule="primary-name">
+    <layout delimiter="; " prefix="(" suffix=")">
+      <choose>
+        <if variable="author editor translator" match="any">
+          <group delimiter=", ">
+            <group delimiter=" ">
+              <text macro="contributors-short"/>
+              <choose>
+                <if match="any" variable="issued">
+                  <date date-parts="year" form="numeric" variable="issued"/>
+                </if>
+                <else>
+                  <text term="no date" form="short"/>
+                </else>
+              </choose>
+            </group>
+            <text variable="locator"/>
+          </group>
+        </if>
+        <else>
+          <group delimiter=", ">
+            <group delimiter=" ">
+              <text macro="title" strip-periods="false"/>
+              <choose>
+                <if variable="issued">
+                  <date variable="issued">
+                    <date-part name="year"/>
+                  </date>
+                </if>
+                <else>
+                  <text term="no date" form="short" font-style="normal"/>
+                </else>
+              </choose>
+            </group>
+            <text variable="locator"/>
+          </group>
+        </else>
+      </choose>
+    </layout>
+  </citation>
+  <bibliography>
+    <sort>
+      <key macro="contributors"/>
+      <key macro="issued-year"/>
+      <key variable="title"/>
+    </sort>
+    <layout suffix=".">
+      <choose>
+        <if variable="author editor translator" match="any">
+          <group delimiter=". ">
+            <group>
+              <text macro="contributors" suffix=" "/>
+              <text macro="issued-year" display="left-margin"/>
```

Comment:
Is the `left-margin` here on purpose? That is very odd and I don't see anything in the author guidelines that'd call for this.